index.js

```diff
@@ -35,5 +35,15 @@ export default async function taskkill(input, options = {}) {
 		arguments_.push(typeof element === 'number' ? '/pid' : '/im', element);
 	}
 
-	await execa('taskkill', arguments_);
+	return arguments_;
 }
+
+export function taskkillSync(input, options = {}) {
+	return execaSync('taskkill', parseArgs(input, options));
```

Comment:
`return` here is not correct. Same with the below.

Reply:
Fixed.